lib/eslint/eslint-helpers.js

```diff
@@ -351,7 +357,7 @@ async function globSearch({
  *      as the user inputted them. Used for errors.
  * @param {Array<string>} options.unmatchedPatterns A non-empty array of absolute path glob patterns
  *      that were unmatched in the original search.
- * @returns {void} Always throws an error.
+ * @returns {Promise<void>} Always throws an error.
```

Comment:
```suggestion
 * @returns {Promise<never>} Always throws an error.
```

This should be the type of `Promise.reject()`.

Reply:
Done!